### ISSUE TITLE
opening stargate for first time using windows 10

### ISSUE THREAD

**Issue by uselessgames** [stargateaudio/stargate#36]
**Describe the bug**
when opening stargate for the first time windows built in "defender smart screen" blocks stargate from opening. Have to manually allow in order to run. Once allowed future instances are not blocked.

**To Reproduce**
try to open stargate, built in windows defender blocks stargate, click more info, allow to run

**Expected behavior**
stargate opens

**Screenshots**
![Annotation 2021-08-08 170043](https://user-images.githubusercontent.com/66875544/128636611-b0f3cfae-110b-4504-8613-9544ab33d5d6.jpg)
![Annotation 2021-08-08 170044](https://user-images.githubusercontent.com/66875544/128636610-937aedfb-dac8-4a6b-8adb-fc4d802b5100.jpg)

**Hardware and operating system**
Intel i5 @1.6GHz, 8GB RAM, laptop, x64, Windows 10 Pro

**Additional context**
n/a

**Comment by stargateaudio** [comment]
Yikes, that is very likely to scare users off.  Thanks for reporting, I will be working feverishly to get this fixed.

**Comment by stargateaudio** [comment]
I spent all day on this, and am not any closer to solving it.  This is a taste of what goes into distributing software on Windows (and Mac) in the year 2021:

https://github.com/daisy/ace-gui/issues/15

I'm going to explore the open source friendly cert providers a bit more, but this is my current level of willingness to support Windows (and Mac, apparently):
1. Spending hundreds of dollars for the privilege of giving away Windows software for free:  Not happy about it, but willing to pay the bribe to the Microsoft "partners".
2. Having my legal name and address exposed to users who download the software, with my name prominently displayed on the scary warning screen:  Completely unwilling to do this, would choose to not support Windows or Mac before doing this
3. Spending thousands of dollars registering a business entity or charity for this project, renting a PO Box that will never receive mail, paying taxes and other legal obligations to avoid disclosing my name and address to the entire world:  At the current stage of the project, no.  If the project had more contributors or received substantial donations or sales, maybe

You would think that the company that owns Github would have considered ways to authenticate open source projects on Github in their own OS without invading developer privacy or taking a 100% cut of $0.00 in revenue and then some.  Frankly, I am so disgusted by this, that I am considering moving the project to Gitlab or Codehut or literally anywhere else before I officially announce and launch the project.  Let's see if this post gets read by robots and flagged as a violation of the terms of service.

TLDR; At this point in time, it's a big maybe, with no timeline.  Unless I can find a reasonable solution, I would be more inclined to drop official Windows support than to be shaken down.  If the app takes off on Linux in a major way and spikes demand, I can always start building it for Windows again, the work is already done.

**Comment by stargateaudio** [comment]
`Certum`(tm) was my last hope for buying a code signing cert.  But:

> The project must be publicly available and **indicate clearly the subscriber’s relationship with the project**. If Certum is unable to identify the project based on publicly available information, the certification application will be rejected.

Once again, Microsoft and partners want to know who you are, and want the entire world to know who you are, if you want to write software for their platform.  At this point I am leaning heavily towards making Stargate Linux-only and moving it to another code hosting website.

**Comment by stargateaudio** [comment]
I evaluated alternative code hosting sites, and there are none:
1. Gitlab:  Almost workable, but 10MB release artifact upload limit.  But that's only a problem if I want to distribute Windows or Mac packages, if I go Linux-only, I probably will
2. Sourceforge:  Yeah, but that is the very definition of metaphorically `Cutting off one's nose to spite their face`, unless I want the street cred of people assuming the project is 20 years old and uses SVN
3. Source Hut: It was a nice thought, but wow they have a long way to go with their web interface
4. Pagure: Maybe, but I would be practically married to Fedora at that point, at a time where IBM / Red Hat is becoming low-key open source hostile, and Fedora is becoming an unstable testing platform for RHEL like the Fedora Core days
